README.md

```diff
@@ -1 +1,89 @@
-# Berry-IMU
\ No newline at end of file
+# Berry-IMU
+This module provides an implementation of the [BerryIMUv3](https://ozzmaker.com/product/berryimu-accelerometer-gyroscope-magnetometer-barometricaltitude-sensor/) using an i2c connection on ARM64 systems.
+
+## Setup
+
+###  Wiring
+Ensure i2c communication is enabled on your single board computer. Wire the SCL and SDA pins on the board to the corresponding pins on the IMU. 
+The IMU can be powered with the 3.3v pin. You can verify the i2c connection by running `i2cdetect -y 1` on your board. If the IMU is wired correctly, you will see 
+the i2c addresses `1c` and `6a`. 
+
+### Registry
+This module can be installed via the viam registry. 
+
+- Go to your robot's page on app.viam.com.
+- Click on the *Create Component* button in the Components section.
+- Search for the *berryimu* module and select it. 
+
+This will automatically install the module to your robot.
+
+
+You can also [run the module locally](https://docs.viam.com/registry/configure/#local-modules).
+
+
+# Calibration
+The BerryIMU's magnetometer needs to be calibrated prior to using this module. Running the calibration script in 'src/calibration.py'
+will provide configuration values to correct for soft and hard iron distortions. 
+
+be sure to have the viam python sdk installed on your machine before running the calibration script:
+```
+pip install viam-sdk
+```
+
+Run the calibration script on your single board computer by calling `python3 calibration.py` in the `src` folder.
```

Comment:
````suggestion
Then, navigate to the `src` directory in the Python SDK, and run the calibration script using the following command:

```sh
python3 calibration.py
```
````

Bumped SBC mention up to "install Py SDK" step.

Reply:
Ah, sorry, you're right: `src` for this module, not the Python SDK. Thanks for catching! 👍

Reply:
no worries!!